tests/analyzer/test_rules.yaml

```diff
@@ -0,0 +1,25 @@
+# SuperBench rules
```

Comment:
Shall we add some yamls for failure configuration. For example. criteria is incorrect, function is incorrect, and metrics are in correct.